portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchUpgrade.testcase

```diff
@@ -19,6 +19,11 @@ definition {
 
 		SearchAdministration.executeSearchActions(actionsDescription = "Reindex all search indexes.");
 
+		ProductMenu.gotoPortlet(
```

Comment:
what is this reason for adding this re-navigation?

Reply:
Because of executeSearchActions macro search Engine information on the Search Engine tab, but the Execute button on Index Actions, so after Executing Reindex all search indexes we need to go back to Search Engine tab or Search portlet then Executing Reindex all spell check indexes. Since on test, they are independent so I prefer to go to search portlet.